### DISCUSSION TITLE
Not Sure if you are interested but this could be useful for some AI tools

### DISCUSSION THREAD

**Discussion by GeekyGhost** [mriale/PyDPainter#133] [Ideas]
So this would actually pair extremely well with comfyUI and stable diffusion. I saw the youtube video you made while looking for tutorials attempting to make my own python based art program for comfyUI. Yours is significantly better than what I have planned and work very well with numerous tools. This would likely be a very popular tool if you decided to port it over. Back to my quest lol! Awesome work!

**Comment by mriale** [comment]
Interesting idea. Would this be for painting something for Img2Img or an inpainting mask? I'm not sure the extra tools in PyDPainter would be necessary to still get an amazing picture out of Stable Diffusion. Today you can draw a picture in PyDPainter, save it, and load it into Img2Img. How would integrating PyDPainter make it better?

**Reply by GeekyGhost** [reply]
I could integrate it with other nodes to make more workflow and tools that shouldn't exists lol. Currently an Krita and Phoroshop/Illustrator type extensions handle the drawing part and there's a limited canvas that is missing a lot of features like primitives and etc that would make it even more useful. There's a gap that many are trying to fill in comfyUI with a tool similar to yours, and yours already exists with the right code structure and etc. It may seem like it's not a big improvement or benefit, but I have plans lol. 

https://civitai.com/models/448101/sprite-sheet-maker

https://civitai.com/models/432889/geeky-ghost-all-in-one-evolved

**Reply by GeekyGhost** [reply]
I'm a mix of techy and artist, so I've been working on building tools for people to use their artistic skills in AI. This would help lol. If you don't, no worries, awesome tool! If you do, looking forward to it! lol no biggie either way. Just thought you work was cool and a good fit :) These are a mix of my Freehand illustrations, Digital art, and 3D models merged into AI works, I'm always on the lookout for tools that let me do more. 

![Screenshot 2024-04-19 234127](https://github.com/mriale/PyDPainter/assets/111990299/3a28d2b5-a625-4293-99f7-e832d3d895eb)
![Screenshot 2024-05-02 205309](https://github.com/mriale/PyDPainter/assets/111990299/e11a7bdf-8092-4708-bf90-cf496d5c583d)

**Comment by mriale** [comment]
I'm not sure I have the time to take PyDPainter in this direction right now. I'm trying to get animation working well on the `develop` branch at the moment. You're free to fork PyDPainter, though, if you want to integrate it into ComfyUI. Just make sure your license is compatible with GPLV3.
